Source/Core/Core/PowerPC/CachedInterpreter/CachedInterpreter.cpp

```diff
@@ -306,95 +242,129 @@ void CachedInterpreter::Jit(u32 address)
     return;
   }
 
-  JitBlock* b = m_block_cache.AllocateBlock(m_ppc_state.pc);
+  if (SetEmitterStateToFreeCodeRegion())
+  {
+    JitBlock* b = m_block_cache.AllocateBlock(em_address);
+    b->normalEntry = b->near_begin = GetWritableCodePtr();
+
+    if (DoJit(em_address, b, nextPC))
+    {
+      // Record what memory region was used so we know what to free if this block gets invalidated.
+      b->near_end = GetWritableCodePtr();
+      b->far_begin = b->far_end = nullptr;
+
+      b->codeSize = static_cast<u32>(GetCodePtr() - b->normalEntry);
```

Comment:
Minor, just to avoid the function call:

```suggestion
      b->codeSize = static_cast<u32>(b->near_end - b->near_begin);
```

Reply:
Sure. This is getting removed in the JIT Widget Refresh though, lol.